lessons/2-shell/slides.qmd

```diff
@@ -1,166 +1,177 @@
 ---
-title: "Shell and Filesystems"
+title: "File systems and the command line"
 subtitle: "Session 2"
-date: "2023-09-07"
+date: "2024-09-10"
 date-format: long
 format: uaz-revealjs
 chalkboard: true
 footer: "[Reproduciblity & Data Science in R](../../index.html)"
 logo: "../../logo.png"
 ---
 
-# Review of last session
+## Review of last session
 
 -   What questions do you have?
 -   What happened as you were trying to re-organize projects or R code?
     -   Victories?
     -   Stumbling blocks?
 
-# Learning objectives
+## Learning objectives
 
--   Understand the what and why of the shell
--   Use the shell to navigate your computer's file system
--   Use R projects to simplify file system navigation
+-   Understand your computer's *file system*
+-   Understand the what and why of the shell command-line interface
+-   Use the CLI to navigate your computer's file system
 -   Know your options for finding help
 
-# What is the shell?
+## How well do you know your file system?
 
--   Command-line interface, in contrast to a GUI
+## Example file system
+
+![](file-path-example.png)
 
-# Why learn the shell?
+## Example file system
+
+![](file-structure-example.png)
 
--   Be able to work with tools that don't have a GUI
--   Batch large tasks (e.g. renaming files)
--   Interact with remote computers like the University HPC
+## How do you navigate your file system?
 
-# Shell in this workshop
+-   Searching
+-   Point and click
+-   "Go to path"
+
+## How do we navigate the file system *programmatically*?
+
+-   A script can't point and click!
+-   We need a system for moving around our file system through written commands.
+
+## What is the shell?
+
+-   Command-line interface, in contrast to a GUI
+-   Useful for navigating the file system
+-   ...and for automating large tasks (e.g. renaming, moving files)
+-   ...*and* for running programs that may not have a GUI
+-   ...**and** for interfacing with remote computers (e.g. the HPC)
+
+## Shell in this workshop
 
 -   git/GitHub (next 2 lessons)
 -   Navigating your file system
 -   Quarto (lesson 9)
 
-# Let's get set up!
+## Let's get set up!
 
 -   macOS comes with a shell (usually Bash or Zsh) built-in
```

Comment:
Just FYI I think zsh has been default for a while (since Catalina which was released in Oct 2019)